R/tax_range_strat.R

```diff
@@ -8,6 +8,8 @@
 #'   `name` and `level` arguments).
 #' @param name \code{character}. The name of the column you wish to be treated
 #'   as the input names, e.g. "genus" (default).
+#' @param group \code{character}. In the case of using "by = "group"", the name
+#' of the column you wish to be treated as the grouping variable, e.g. "family".
```

Comment:
I think this should be right after the definition of "by". Also, is this argument necessary? If the user supplies something other than "FAD", "LAD", or "name", we could just assume that is the name of the other variable?